.github/workflows/main.yml

```diff
@@ -350,7 +346,7 @@ jobs:
     - name: Start background ganache
       run: |
         cd proxy
-        NODE_OPTIONS="--max_old_space_size=3072" npx ganache-cli --gasLimit 12000000 --quiet --allowUnlimitedContractSize --acctKeys ../test/accounts.json &
+        NODE_OPTIONS="--max_old_space_size=3072" npx ganache-cli --gasLimit 0xfffffffffff --quiet --allowUnlimitedContractSize --defaultBalanceEther 2000000 --account="0x23ABDBD3C61B5330AF61EBE8BEF582F4E5CC08E554053A718BDCE7813B9DC1FC,1000000000000000000000000000000000" --account="0x80ebc2e00b8f13c5e2622b5694ab63ee80f7c5399554d2a12feeb0212eb8c69e,1000000000000000000000000000000000" --acctKeys ../test/accounts.json &
```

Comment:
Why do you need to specify private keys?
You can get first 2 from the accounts.json and pass them to tests or in tests get first 2 from the accounts.json

Reply:
Now using `--gasLimit 12000000` again. Fixed.

Reply:
We are using command line private keys because I did not found way to insert these two keys into `../test/accounts.json` file.

Reply:
I see that `gasLimit` is still set to `0xfffffffffff`

Reply:
> We are using command line private keys because I did not found way to insert these two keys into `../test/accounts.json` file.

ganache does it for you because of `acctKeys` flag.
From ganache documentation:
```
--account_keys_path, --acctKeys
saves generated accounts and private keys as JSON object in specified file
```

Reply:
Latest code is `NODE_OPTIONS="--max_old_space_size=3072" npx ganache-cli --gasLimit 12000000 --quiet --allowUnlimitedContractSize --defaultBalanceEther 2000000 --acctKeys ../test/accounts.json &`

Reply:
So, you commented obsolete version.

Reply:
Done. Rolled back to `--gasLimit 12000000`. Command line private keys removed. It works.